app/models/patient.rb

```diff
@@ -25,23 +25,20 @@ class Patient < ApplicationRecord
 
   validate :valid_birth_date
 
-  after_initialize :set_main_ubs
```

Comment:
after_initialize was making a call to Ubs every time there as a Pacient object in pure Ruby. Changing this to before_validate will save at least 1 `SELECT * FROM ubs` on every single request 🥳 .